Bowling.java

```diff
@@ -21,7 +22,10 @@ public Integer score() {
 	}
 
 	public String printScoreboard() {
-		return "";
+		return scoreboard.getFrames().stream()
+			.filter(Frame::isFull)
+			.map(Frame::printFrame)
+			.collect(Collectors.joining(","));
```

Comment:
I almost went with the return the extract string that was asked for, but decided that showing off some java 8 streaming might be better.  I'm not positive about `Frame.printFrame()` - seems like that might need a better home and not give the Frame knowledge of how it should be represented.  Maybe just a method in the Bowling class would be better...